src/main/java/com/petros/bibernate/session/Persistence.java

```diff
@@ -1,5 +1,18 @@
 package com.petros.bibernate.session;
 
+/**
+ * The entry point of the application. Persistence creates a SessionFactory.
+ * Example:
+ * <pre>{@code
+ *      ...
+ *      public static void main(String[] args) {
+ *          SessionFactory sessionFactory = Persistence.createSessionFactory();
```

Comment:
createSessionFactory without parameters will take parameters to create datasource from properties file. Later, when we start implementing multi-databases support, we will overload this method by adding "persistenceUnit" parameter. I think we should discuss the properties file structure with persistenceUnits.

Reply:
new task has been created https://trello.com/c/aPH9hYbY/30-implement-multi-database-support

Reply:
Yes, but my concern is that it would be better to pass a datasource created by the user to this method at the initial stage. This will eliminate the incorrect dependency on H2 inside the SessionFactoryImpl class, and in the future, we can remove it when we are able to pass persistenceUnit.

Reply:
My next PR will contain datasource creation based on properties from file automatically. This implementation does not depend on specific datasource. We can override the method and pass there datasource but I do not see any purpose to add something that will be deleted in the future.